addons/mail/static/tests/message/message_reply.test.js

```diff
@@ -120,3 +121,36 @@ test("click on message in reply highlights original message", async () => {
     );
     await contains(".o-mail-Message.o-highlighted:contains('This message has been removed')");
 });
+
+test("can reply to logged note in chatter", async () => {
+    const pyEnv = await startServer();
+    const partnerBId = pyEnv["res.partner"].create({ name: "Partner B" });
+    pyEnv["mail.message"].create([
+        {
+            author_id: partnerBId,
+            body: "Test message from B",
+            model: "res.partner",
+            res_id: serverState.partnerId,
+            subtype_id: pyEnv["mail.message.subtype"].search([
+                ["subtype_xmlid", "=", "mail.mt_note"],
+            ])[0],
+        },
+        {
+            author_id: serverState.partnerId,
+            body: "Another msg",
+            model: "res.partner",
+            res_id: serverState.partnerId,
+            subtype_id: pyEnv["mail.message.subtype"].search([
+                ["subtype_xmlid", "=", "mail.mt_note"],
+            ])[0],
+        },
+    ]);
+    await start();
+    await openFormView("res.partner", serverState.partnerId);
+    await click(".o-mail-Message:contains('Test message from B') button[title='Reply']");
+    await contains("button.active", { text: "Log note" });
+    await contains(".o-mail-Composer.o-focused .o-mail-Composer-input", { value: "@Partner B " });
+    await click(".o-mail-Composer button", { text: "Log" });
+    await contains(".o-mail-Message a.o_mail_redirect", { text: "@Partner B" });
+    await contains(".o-mail-Message:contains('Another msg') button[title='Reply']", { count: 0 });
```

Comment:
Asserting Reply is not in quick action of Message doesn't mean the action is not present at all: it could be in the "expand" / "..." part of the message.
So in practice should check the reply feature is not present there. A negative assertion should have a positive assertion using similar selector for decent robustness of test, so we could setup a message in channel and check there's the reply feature as dropdown item, so that here we can check it's not in the dropdown.

Note that checking `count: 0` should be done after checking actual presence of another dropdown item, otherwise the `contains({count:0})` could pass because it checks before the dropdown opens (and renders the items, it's ok to assume if 1 item is rendered then all items are rendered).